gr-digital/lib/header_payload_demux_impl.cc

```diff
@@ -361,14 +381,24 @@ int header_payload_demux_impl::find_trigger_signal(int skip_items,
                                                    const unsigned char* in_trigger)
 {
     int rel_offset = max_rel_offset;
+    int trigger_nums = 0;
     if (max_rel_offset < skip_items) {
         return rel_offset;
     }
     if (in_trigger) {
+        for (int i = skip_items; i < max_rel_offset; i++) {
```

Comment:
Is this a duplicate line?

Reply:
Oh! yes, I didn't check it carefully,i'm sorry